kpatch-build/lookup.c

```diff
@@ -84,7 +84,9 @@ static int maybe_discarded_sym(const char *name)
 	    !strncmp(name, "__brk_reservation_fn_", 21) ||
 	    !strncmp(name, "__func_stack_frame_non_standard_", 32) ||
 	    !strncmp(name, "__addressable_", 14) ||
-	    !strncmp(name, "__UNIQUE_ID_", 12))
+	    !strncmp(name, "__UNIQUE_ID_", 12) ||
+	    !strncmp(name, ".L.str", 6) ||
+	    strstr(name, ".__UNIQUE_ID_"))
```

Comment:
This `trace_nfsd_exp_get_by_name.__UNIQUE_ID___addressable___SCK__tp_func_nfsd_exp_get_by_name645` thing is mysterious, I can't figure out where that comes from.  Investigating...

Anyway, here we can probably just broaden the existing unique/addressable checks to `strstr(name, "__UNIQUE_ID_")` and `strstr(name, "__addressable_")`.

Reply:
BTW, this function is a bit crude, it might be smarter to check to see if the symbol is in a ".discard.* section.  That may eliminate some or all of these checks.  (not a requirement for this PR, just an idea)

Reply:
It wouldn't eliminate the `.L.str` check but may get rid of some of the others.